City-of-Helsinki/drupal-helfi-kymp
# UHF-2291: Purge queue cron, purge config and dependencies

To test this, run:

- `composer install`
- `drush cr && drush cim`
- Create `public/sites/default/local.settings.php` file and add:
```
$config['system.performance']['cache']['page']['max_age'] = (86400 * 32);
$config['varnish_purger.settings.default']['hostname'] = 'helfi-kymp-varnish';
```

1. Open any URL through Varnish as an anonymous user, like: https://varnish-helfi-kymp.docker.so/fi/kaupunkiymparisto-ja-liikenne/payment-methods-and-parking-fine
2. Check the response headers and make sure the `age` header is higher than 0:
![2021-09-17-10-18-24-screenshot](https://user-images.githubusercontent.com/771113/133741178-1f0e2de0-3458-4185-99e0-b9b9727012c5.png)
3. Edit the content you just visited and run `drush cron`
4. Make sure the content is updated
 
If you have any issues, you can try to rebuild containers with `docker-compose build` and then restarting them `make stop && make up`.
